xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -1957,7 +1963,11 @@ async fn apply_update_group_membership_intent(
 
     // Update the extensions to have the new GroupMembership
     let mut new_extensions = extensions.clone();
-    new_group_membership.failed_installations = changes_with_kps.failed_installations;
+    new_group_membership.failed_installations = [
```

Comment:
Just a high level question - are we considering the failed installations that were in this field before this line?

Reply:
yes, we're merging the new failed_installations and the old ones.

Reply:
fyi: if we have before two failed installations, and now one new failed_installation, we need to keep the track of both.